moveit_plugins/moveit_ros_control_interface/CMakeLists.txt

```diff
@@ -76,11 +76,7 @@ EXPORT export_${PROJECT_NAME}
 ARCHIVE DESTINATION lib
 LIBRARY DESTINATION lib
 RUNTIME DESTINATION bin
-)
-
-## Mark cpp header files for installation
-install(DIRECTORY include/${PROJECT_NAME}/
```

Comment:
Usually, we would use `install(DIRECTORY include/ DESTINATION include)`. Otherwise, the headers will not be inside the projects path when you call `#include <project_name/header.h>`

Reply:
OK, I'll revert the change that deleted this line and push up again. I didn't do a clean build for this package when I was testing my change, which I realized means that changes to what was installed weren't reflected by the state of the built workspace.